Sources/Foundation/FileHandle.swift

```diff
@@ -310,9 +310,15 @@ open class FileHandle : NSObject {
                 let data = mmap(nil, mapSize, PROT_READ, MAP_PRIVATE, _fd, 0)
                 // Swift does not currently expose MAP_FAILURE
                 if data != UnsafeMutableRawPointer(bitPattern: -1) {
+                  #if os(Android)
+                    return NSData.NSDataReadResult(bytes: data, length: mapSize) { buffer, length in
+                        munmap(buffer, length)
+                    }
+                  #else
                     return NSData.NSDataReadResult(bytes: data!, length: mapSize) { buffer, length in
```

Comment:
As shown in my Bionic link above, `mmap()` now returns `_Nonnull`, so this force-unwrap fails.

Reply:
It feels rather unfortunate to ifdef around this here instead of at the place we call `mmap`.